bin/summit.py

```diff
@@ -1295,6 +1306,8 @@ def usage():
                           (sets ack point = read point)
       --nack              negative acknowledgement for events read
                           (sets read point = ack point)
+      --flush             discard all existing events from the event 
```

Comment:
suggest something like: discard unread events
                                     (set read and ack point to end of channel)